frontend/src/lib-customizations/defaults/citizen/i18n/fi.tsx

```diff
@@ -2573,7 +2573,7 @@ export default {
     placementTermination: {
       title: 'Paikan irtisanominen',
       description:
-        'Irtisanoessasi paikkaa huomaathan, että mahdollinen siirtohakemus poistuu viimeisen läsnäolopäivän jälkeen. Jos tarvitset lapsellesi myöhemmin paikan, sinun tulee hakea sitä uudella hakemuksella.',
+        'Irtisanoessasi paikkaa huomaathan, että mahdollinen siirtohakemus poistuu viimeisen läsnäolopäivän jälkeen. Jos tarvitset lapsellesi myöhemmin paikan, sinun tulee hakea sitä uudella hakemuksella. Tallennathan kaikki dokumentit, jotka haluat säilyttää, ennen lapsen viimeistä läsnäolopäivää. Voit tallentaa pedagogiset asiakirjat ja dokumentit omiin tiedostoihisi lataustoiminnolla.',
```

Comment:
Eka uusi virke on jotenkin kankea kahdella pilkullaan. Olisiko esim: `Muista tallentaa kaikki haluamasi asiakirjat ennen lapsesi viimeistä läsnäolopäivää.`?

Reply:
Samaa mieltä mutta nämä tuli tuotehallinnalta käännöksineen niin laitan tämän nyt eteenpäin ja korjataan lisää jos ja kun pyyntö tulee